dashboard/app/controllers/pd/workshop_user_admin_controller.rb

```diff
@@ -0,0 +1,41 @@
+class Pd::WorkshopUserAdminController < ApplicationController
+  authorize_resource class: :pd_workshop_user_admin
+
+  # display the Facilitator search form, Facilitators found, and Courses for first Facilitator found
+  # assign / remove Courses actions redirect back to this form to display the updated course list
+  def facilitator_courses_form
+    search_term = facilitator_course_params[:search_term]
+    if search_term =~ /^\d+$/
+      facilitator_id = search_term
+      @facilitators = User.joins(:permissions).where(id: facilitator_id, user_permissions: {permission: UserPermission::FACILITATOR})
```

Comment:
Our [styleguide](https://github.com/code-dot-org/code-dot-org/blob/staging/STYLEGUIDE.md#all-languages) recommends 80 cols, with a max of 120. Please split up this long line, e.g.

``` 
@facilitators = User.joins(:permissions).where(
  id: facilitator_id, 
  user_permissions: {permission: UserPermission::FACILITATOR}
)
```